core/src/main/java/com/novoda/noplayer/drm/provision/ProvisionExecutorImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.novoda.noplayer.drm.provision;
+
+import com.novoda.noplayer.drm.ModularDrmProvisionRequest;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+class ProvisionExecutorImpl implements ProvisionExecutor {
```

Comment:
Can we call this something that doesn't end in `impl`? `HttpPostingProvisionExecutor` or something maybe?